src/prerecorded/options/mod.rs

```diff
@@ -0,0 +1,337 @@
+use serde::{ser::SerializeSeq, Serialize};
+
+#[derive(Debug)]
+pub struct OptionsBuilder<'a> {
+    model: Option<Model<'a>>,
+    version: Option<&'a str>,
+    language: Option<Language>,
+    punctuate: Option<bool>,
+    profanity_filter: Option<bool>,
+    redact: Vec<Redact>,
+    diarize: Option<bool>,
+    ner: Option<bool>,
+    multichannel: Option<bool>,
+    alternatives: Option<usize>,
+    numerals: Option<bool>,
+    search: Vec<&'a str>,
+    callback: Option<&'a str>,
+    keywords: Vec<&'a str>,
+    utterances: Option<Utterances>,
+    tag: Option<&'a str>,
+}
+
+#[derive(Debug)]
+pub enum Model<'a> {
+    General,
+    Meeting,
+    Phonecall,
+    Voicemail,
+    Finance,
+    Conversational,
+    Video,
+    CustomId(&'a str),
+}
+
+#[derive(Debug, Clone, Copy)]
+#[non_exhaustive]
+#[allow(non_camel_case_types)]
+pub enum Language {
+    zh_CN,
+    zh_TW,
+    nl,
+    en_US,
+    en_AU,
+    en_GB,
+    en_IN,
+    en_NZ,
+    fr,
+    fr_CA,
+    de,
+    hi,
+    id,
+    it,
+    ja,
+    ko,
+    pt,
+    pr_BR,
+    ru,
+    es,
+    es_419,
+    sv,
+    tr,
+    uk,
+}
+
+#[derive(Debug)]
+pub enum Redact {
+    Pci,
+    Numbers,
+    Ssn,
+}
```

Comment:
Not as worried about the closed set of redactions, as that doesn't change as frequently as supported languages.